components/src/Tables/Table.js

```diff
@@ -1,38 +1,98 @@
 import React from 'react';
 import styled from 'styled-components';
+import { colour, space, font, radius, shadow } from '@nulogy/tokens';
+import QuietButton from '../Button/QuietButton';
 
-const borderStyle = ({ theme }) => `1px solid ${theme.colour.neutral[400]}`;
+const borderWidth = 1;
+
+const withoutBorder = value => value - borderWidth;
+
+const lineHeight = name => font.lineHeight[name] * font.size.medium;
 
 export const Cell = styled.td`
-  padding: 1rem;
-  text-align: left;
-  border-bottom: ${borderStyle};
+  font-size: ${font.size.small}px;
+  line-height: ${lineHeight('smaller') / font.size.small};
+  padding: ${space.x2}px ${space.x1}px;
+  &:first-child{
+    padding-left: ${space.x2}px;
+  }
+  &:last-child{
+    padding-right: ${space.x2}px;
+    width:72px; // not the right place
+  }
 `;
 
-export const Row = styled.tr``;
+export const ActionCell = styled(Cell)`
+  padding: ${space.x1}px ${space.x2}px ${space.x1}px ${space.x1}px;
+  display: flex;
+  justify-content: flex-end;
+  & > button:not(:last-child) {
+    margin-right: ${space.half}px;
+  }
+`;
 
-export const Header = styled.thead``;
+export const Row = styled.tr`
+  &:nth-child(odd){
+    background-color: ${colour.blue['200']};
+  }
+`;
 
-export const Body = styled.tbody``;
+export const Header = styled.thead`
+  text-align: left;
+`;
 
 export const HeaderCell = styled(Cell.withComponent('th'))`
   font-weight: bold;
-  border-bottom-width: 2px;
+  background-color: ${colour.white}; // sloppy
+`;
+
+export const CreateRow = styled.tr`
+  background-color: ${colour.blue[800]};
+`;
+
+export const CreateRowCell = styled(HeaderCell.withComponent('th'))`
```

Comment:
1. I am surprised that his is a `HeaderCell`, why not a `td`?
2. `HeaderCell` is already a `th`, no need to call `withComponent` again.